app/src/test/java/org/mozilla/tv/firefox/ext/LocaleKtTest.kt

```diff
@@ -62,4 +67,45 @@ class LocaleKtTest {
 
         assertTrue(Locale.US.languageAndMaybeCountryMatch(arrayOf(Locale.CANADA, allowed)))
     }
+
+    @Test
+    fun `GIVEN all killswitch locale is allowed WHEN any locale is passed THEN should return true`() {
+        val allowed = KillswitchLocales.All
+
+        assertTrue(Locale.US.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.CANADA.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.UK.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.GERMANY.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.CHINA.languageAndMaybeCountryMatch(allowed))
+
+        assertTrue(Locale.ENGLISH.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.GERMAN.languageAndMaybeCountryMatch(allowed))
+        assertTrue(Locale.CHINESE.languageAndMaybeCountryMatch(allowed))
+    }
+
+    @Test
+    fun `WHEN KillswitchLocale ActiveIn is passed THEN overload should be called`() {
+        // Mock extension functions in this class
+        // If this test fails, check if this class has been moved, causing this string to be incorrect
+        mockkStatic("org.mozilla.tv.firefox.ext.LocaleKt")
```

Comment:
nit: We can put the string in a constant at the top of the file.

Thanks for the handy comment but unfortunately if we ever call this function again, we'll be duplicating it a lot. :\

Reply:
Yeah, unfortunately true.  I'm not 100% happy with this one, but didn't see a better way to do it.